### ISSUE TITLE
Data Processing Pipeline

### ISSUE THREAD

**Issue by XiuLuoShen** [Bot-FBI/Issue-Tracker#8]
Develop pipeline to process data (accounts and tweets) and convert into pytorch tensors that can be loaded for training.

Requirements:
- [ ] Vocabulary/embedding for text (Currently basic english letters. Not started)
- [x] Dataloader that can be used to batch data (account and tweets) together and generate pytorch tensors which can be fed into the model
- [ ] Dataloader that join multiple tweets to an account together and generated pytorch tensors which can be fed into the model

TODO: Optimize code

**Comment by XiuLuoShen** [comment]
Data Loader in PyTorch created with padding and batching for text.
